common-healing.lic

```diff
@@ -347,8 +348,9 @@ module DRCH
 
   # Skill check to tend a bleeding wound.
   # If you're not skilled enough you may make things worse and incur long roundtime.
-  def skilled_to_tend_wound?(bleed_rate)
-    min_skill = $DRCH_BLEED_RATE_TO_SEVERITY_MAP[bleed_rate][:skill_to_tend]
+  def skilled_to_tend_wound?(bleed_rate, internal = false)
```

Comment:
Why pass in `internal`? I.e. why not call `internal?` directly inside this function?

Reply:
Good question. It's because skill_to_tend is a method of DRCH and internal is a method of the inner class Wound, which happens to be defined in DRCH too, but are different lexical scopes.

Reply:
Makes sense 👍